### PR TITLE
Issue with views and system tables

### PR DESCRIPTION
**Pull request by alc**
The [current query](https://github.com/atlasphp/Atlas.Info/blob/1.x/src/Info.php#L39) to retrieve table names from the database ignores the fact that `information_schema.tables` also contains information for views and system tables, which Atlas.Orm doesn't understand.  This pull request adds a where clause to the query to ensure that only the names of user-defined tables are retrieved.

See information about TABLE_TYPE column in e.g. the [MySQL docs](https://dev.mysql.com/doc/refman/5.7/en/tables-table.html), for example.

### CODE REVIEW

**Comment by alc** [comment]
**tl;dr** It would be cool if Atlas.Orm could understand views and use them like readonly tables.

Some details on how I discovered this problem:

I have a database with a view in it and ran the skeleton generator to see what would happen.  All the expected classes for the view were created, and I was cautiously optimistic that I would be able to fetch related records via the view (my view looks like a many-to-many table).  That didn't work because the serialization for rows in the view was wrong - the "table" had no primary key(s), so each row was serialized as the same.

I know the answer to this statement is "open a pull request", but it would be cool if Atlas.Orm could understand views and use them like readonly tables.

Thanks for all your hard work, @pmjones

**Comment by pmjones** [comment]
@alc Thanks for this fix! Re: views etc., yeah, opening a PR would be best, but opening a separate issue would be second-best; add as much detail and info as you can for your situation, the problems you found, and how they *might* be addressed -- all that will help me or someone else write up a solution. 

Again, my thanks for your work here!

**Comment by pmjones** [comment]
This causes the PgsqlInfoTest::testFetchTableNames() test to fail; any thoughts here?

**Comment by pmjones** [comment]
Ah, it is that Postgres keeps the values in upper-case. Easy fix!
